usr/share/rear/prep/default/310_include_uefi_tools.sh

```diff
@@ -8,7 +8,9 @@ if [[ ! -d /boot/[eE][fF][iI] ]]; then
     if is_true $USING_UEFI_BOOTLOADER; then
         Error "USING_UEFI_BOOTLOADER = 1 but there is no directory at /boot/efi or /boot/EFI" # abort
     fi
-    return # skip
+    if ! grep -q '/boot/efi' /proc/mounts; then
```

Comment:
I think that using `mountpoint  -q /boot/efi` is more elegant.

Reply:
or rather `mountpoint  -q /boot/[Ee][Ff][Ii]` as we want to be case-insensitive.